app/assets/src/components/views/discovery/discovery_sidebar.scss

```diff
@@ -32,11 +33,18 @@
 
     color: $black;
     font-weight: 600;
-    letter-spacing: 1.45px;
+    font-size: 12px;
+    letter-spacing: 1.3px;
     text-transform: uppercase;
     padding: 0 10px;
-    height: 50px;
-    line-height: 50px;
+    height: 40px;
```

Comment:
any thoughts about combining this CSS with that of `SampleDetailsMode` because it supposed to look the same? what's the best way to abstract it?

Reply:
I combined these two .title into `@include font-header-tall-section-title` because they were identical